Bug Issue Template.md

```diff
@@ -0,0 +1,57 @@
+## Story
+**As a <Agent, Admin, or Patient> of <STDcheck, Healthlabs, UTITreatment and/or Manage>**, I need < an expression of the correct behavior needed from the system >.
+
+## Steps to Recreate  
+
+< a numbered list of the **exact** steps taken in the system that led to the observation of the bug. screen shots or videos can help >
```

Comment:
This still says "observe the bug" rather than "recreate the bug":

```suggestion
< a numbered list of the **exact** steps to take in the system to re-create the bug. screen shots or videos can help >
```

Reply:
I went a little beyond the suggestion. I think the extra sentence will help non-Devs create Issues better. We probably need additional information in the form they fill out.

Reply:
Even better 👍